src/Kopernicus/Configuration/SpaceCenterLoader.cs

```diff
@@ -415,6 +419,7 @@ public Vector2Parser EditorGroundTexOffsetParser
         // Apply event
         void IParserEventSubscriber.Apply(ConfigNode node)
         {
+            KSC.Instance = Value;
```

Comment:
This shouldn't be neccessary. `KSC.Instance` is set when the loading was completed (to whatever space center was found on the home body) and while loading all calls to it are guarded with `if (!Injector.IsPrefab)` checks.

EDIT: I see the problem, you are storing everything in the `KSC` object, and that needs to be passed through to the `GrassMaterialLoader`. I think this could be solved by creating the `GrassMaterialLoader` ourselves in the constructor, instead of letting the parser do it. Then the `KSC` object could be passed through the constructor.

I just don't like the parser modifying global states, unless it is absolutely neccessary.

Reply:
I'll do that if you prefere

Reply:
I think this is fixed now, I'll leave the PR open for you to review